andela/wger_asgards
# [#137085745] - Build REST api Endpoints for adding and viewing Users

#### What does this PR do?
Add and View users endpoint on wger's REST API

#### Description of tasks to be completed?
Have the following endpoints working
`GET /api/v1/users/`
`POST /api/v1/users`
`GET /api/v1/users/<users_id>`

#### How should this be manually tested?
After cloning the repo and following the step by step procedures on the readme, make sure your project can run locally and now run `python manage test users`
- Using Postman test every endpoint with the above header:
_key:_ Content-type _value:_ application/json 

- To test this authentication, add this to the header: Get TOKEN from login endpoint
_key:_ Authorisation _value:_ JWT TOKEN

**Please note** : Wger has a live browsing view which is an alternative to test the endpoints above. 

#### Any background context you want to provide?
There's some implementation on tastypie  which is deprecated in version 1.9 for performance optimisation.

#### What are the relevant pivotal tracker stories?
#137085745
